src/OpenLoco/Objects/ObjectManager.h

```diff
@@ -114,56 +114,6 @@ namespace OpenLoco::ObjectManager
     constexpr size_t maxObjects = 859;
```

Comment:
unrelated to this PR, but these should be prefixed with `k` right?

Reply:
I'd like to remove this constant altogether